101-app-service-regional-vnet-integration/azuredeploy.json

```diff
@@ -0,0 +1,96 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "appName": {
+            "type": "string"
+        },
+        "appServicePlanName": {
+            "type": "string"
```

Comment:
add a defaultValue that will be unique within the subscription

Reply:
Done.